docs/music-server.md

```diff
@@ -361,3 +415,145 @@ DocumentRoot "C:\Bemuse\myserver"
 Once the file is saved, open your XAMPP Control Pannel and run "Apache".
 
 Then connect to the music server with (http://bemuse.ninja/?server=https://localhost/).
+
+### On a Local Machine (macOS, [nginx](https://nginx.org/en/))
+
+First, open a Terminal and `cd` into the folder that contains the Bemuse server (there should be an `index.json` file in that folder).
+
+Then, install nginx:
+
+```bash
+$ brew install nginx
+```
+
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>
+    Homebrew will install nginx on /usr/local.
+</p>
+</div>
+
+Check your current location:
+
+```bash
+$ pwd
+# e.g Output: /Users/cenox/Dev/bemuse-test
+```
+Note the output location; this will be used when we configure nginx.
+
+Next, `cd` to nginx folder:
+
+```bash
+$ cd /usr/local/nginx/etc
+```
+
+Use `touch` command to create a config file to serve the Bemuse server.
+
+```bash
+$ touch servers/bemuse
+```
+
+Open Bemuse setting file with a text editor. Our example will use `nano`:
+
+```bash
+$ nano servers/bemuse
+```
+
+Insert below text:
+
+```nginx
+server {
+    listen 80;
+    location / {
+        add_header 'Access-Control-Allow-Origin' '*';
+        root /Users/cenox/Dev/bemuse-test;
+        #    ^ Replace with the location from previous step
+    }
+}
+```
+
+Save the file by pressing `Ctrl+X`, `Y`, `Enter`.
+
+Reload nginx:
+
+```bash
+$ nginx -s reload
+```
+
+Now, the music server will be available at http://localhost. You can play by going to <http://bemuse.ninja/?server=http://localhost>.
+
+##### Additional Note.
+
+When you install nginx in Linux using a package manager, it runs when the OS boots up, but macOS isn't.
+You can use `brew services` to register the nginx service and make nginx automatically start when you log in.
+
+```bash
+$ brew services start nginx
+```
+
+#### Debian Linux using [nginx](https://nginx.org/en/)
+
+First, open a Terminal and `cd` into the folder that contains the Bemuse music server (you should see `index.json` in this folder).
+
+Then, install nginx:
+
+```bash
+$ sudo apt install nginx
+```
+
+Check your current location:
+
+```bash
+$ pwd
+# e.g Output: /home/cenox/bemuse-test
+```
+
+Note the output location. This will be used when we configure nginx.
+
+Go to nginx folder:
+
+```bash
+$ cd /etc/nginx
+```
+
+Use `touch` command to create a config file to serve the Bemuse music server.
+
+```bash
+$ sudo touch site-available/bemuse
+```
+
+Open the setting file with your text editor. This example uses `nano`:
+
+```bash
+$ sudo nano servers/bemuse
+```
+
+Insert below text:
+
+```nginx
+server {
+    listen 80;
+    server_name localhost;
+    location / {
+        add_header 'Access-Control-Allow-Origin' '*';
+        root /home/cenox/bemuse-test; # Write location from pwd command.
+    }
+}
+```
+
+Save the file by pressing `Ctrl+X`, `Y`, `Enter`.
+
+Then, create a symlink to enable the configured site:
+
+```bash
+$ sudo ln -s /etc/nginx/site-available/bemuse site-enabled/bemuse
```

Comment:
Are you sure it’s called `site-available` and `site-enabled`, and not `sites-available` and `sites-enabled`?

Reply:
Oh you right, it is site's' available and enabled. I'll commit changes.